packages/build/tests/core/tests.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-magic-numbers */
```

Comment:
Should this be moved around the `Passes the right feature flags to zip-it-and-ship-it` test instead as `eslint-disable` then `eslint-enable`?

Reply:
I was actually thinking of disabling this rule for all test files. Would you be open to that?

Reply:
Yes, this works :+1:

Reply:
Done in https://github.com/netlify/build/pull/3925/commits/aefe7b32cca7a943e0b4a3dd0b1423b4ac787df5.